test/junit/scala/OptionTest.scala

```diff
@@ -22,8 +22,8 @@ class OptionTest extends RunTesting {
   import runner._
   import scala.tools.reflect.{ ToolBoxError => E }
 
-  @Test(expected = classOf[E]) def testSomeZipList(): Unit = run("""Some("foo") zip List("bar", "baz")""")
-  @Test(expected = classOf[E]) def testSomeZipNil(): Unit = run("""Some("foo") zip Nil""")
-  @Test(expected = classOf[E]) def testNoneZipList(): Unit = run("""None zip List("bar")""")
-  @Test(expected = classOf[E]) def testNoneZipNil(): Unit = run("""None zip Nil""")
+  @Test() def testSomeZipList(): Unit = assertThrows(classOf[E], run("""Some("foo") zip List("bar", "baz")"""))
+  @Test() def testSomeZipNil(): Unit = assertThrows(classOf[E], run("""Some("foo") zip Nil"""))
+  @Test() def testNoneZipList(): Unit = assertThrows(classOf[E], run("""None zip List("bar")"""))
+  @Test() def testNoneZipNil(): Unit = assertThrows(classOf[E], run("""None zip Nil"""))
```

Comment:
why is this not `() => run(...)`?

Reply:
(note, this question ends up being moot if we switch to `scala.tools.testkit.AssertUtil.assertThrows`, if that happens)